source/switching_to_live/index.html.md.erb

```diff
@@ -134,7 +134,7 @@ You make a real payment using either:
 
 1. After you've confirmed the payment, check you received a [confirmation email](/payment_flow/#confirmation-email).
 
-1. Check the payment appears in [your transactions](https://selfservice.payments.service.gov.uk/all-service-transactions) in the GOV.UK Pay admin tool.
+1. Check the payment appears in [your live transactions](https://selfservice.payments.service.gov.uk/all-service-transactions/live) in the GOV.UK Pay admin tool.
```

Comment:
do we want to link to all transactions or to the specific service transactions? or say that you can see it in both...?